### DISCUSSION TITLE
We need an upgrade page in the docs

### DISCUSSION THREAD

**Discussion by amychen1776** [dbt-labs/docs.getdbt.com#986] [Archive]
Right now, when you're using an outdated version of dbt, the message that you get directs you to the installation page which no longer has a page on what commands you need to upgrade your adapter version to the latest core version. 

![image](https://user-images.githubusercontent.com/46451573/147588515-e4a18a35-1f4b-444b-8650-e26f22e7e544.png)

**Comment by runleonarun** [comment]
I agree! @jasnonaz started a [PR here](https://github.com/dbt-labs/docs.getdbt.com/pull/977), but I was hoping we could create a page for upgrades instead of just a section. What do you both think?
